examples/src/main/java/com/jagrosh/jdautilities/examples/command/AboutCommand.java

```diff
@@ -15,107 +15,205 @@
-public class AboutCommand extends Command {
-    private boolean IS_AUTHOR = true;
-    private String REPLACEMENT_ICON = "+";
+@Author(" John Grosh (jagrosh)")
+public class AboutCommand extends Command
+{
+
     private final Color color;
     private final String description;
-    private final Permission[] perms;
-    private String oauthLink;
-    private final String[] features;
-    
-    public AboutCommand(Color color, String description, String[] features, Permission... perms)
+    private final Command[] commands;
+    private Permission[] perms;
+    private String inviteLink;
+
+    /**
+     * Initialize an About command to send information about your bot!
+     *
+     * @param color The Embed Color
+     * @param description The bot's Description
+     * @param commands All the commands to show
+     * @param invPerms Permissions to generate an Invite Link
+     */
+    public AboutCommand(Color color, String description, Command[] commands,
+                        Permission... invPerms)
     {
-        this.color = color;
-        this.description = description;
-        this.features = features;
-        this.name = "about";
+        //Super variables
+        this(color, description, commands);
+
+        //About Command variables
+        this.perms = invPerms;
+        this.inviteLink = null;
+    }
+
+    /**
+     * Initialize an About command to send information about your bot!
+     *
+     * @param color The Embed Color
+     * @param description The bot's Description
+     * @param commands All the commands to show
+     * @param inviteLink The invite link for the bot
+     */
+    public AboutCommand(Color color, String description, Command[] commands,
+                        String inviteLink)
+    {
+        //Super variables
+        this(color,description, commands);
+
+        //About Command variables
+        this.inviteLink = inviteLink;
+        this.perms = null;
+    }
+
+    /**
+     * The default constructor, this is used to clean the clutter from other constructors
+     *
+     * @param color The Embed Color
+     * @param description The bot's Description
+     * @param commands All the commands to show
+     */
+    private AboutCommand(Color color, String description, Command[] commands)
+    {
+        //Super variables
+        this.name = "About";
+        this.aliases = new String[] {"whoareyou"};
         this.help = "shows info about the bot";
         this.guildOnly = false;
-        this.perms = perms;
         this.botPermissions = new Permission[]{Permission.MESSAGE_EMBED_LINKS};
+
+        //About Command variables
+        this.color = color;
+        this.description = description;
+        this.commands = commands;
     }
-    
-    public void setIsAuthor(boolean value)
+
+
+    @Override
+    protected void execute(CommandEvent event)
     {
-        this.IS_AUTHOR = value;
+        //Get info about the bot
+        event.getJDA().asBot().getApplicationInfo().queue(info -> {
```

Comment:
Using `queue` here is rather unnecessary, as the version with `complete` only calls it once: the very first time the command is run. We also don't want to fetch this info every time the command is run, since it rarely changes.